OxfordBodleian/Juel-Jensen/BDLaethf29.xml

```diff
@@ -165,62 +513,345 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                    <ab type="pricking">Ruling pricks are mostly visible.</ab>
+                                    <ab type="ruling">The upper line is written above the
+                                        ruling.</ab>
+                                    <ab type="ruling">The bottom line is written above the
+                                        ruling.</ab>
+                                    <ab type="ruling"> The ruling pattern is clearly different on
+                                            <locus from="121v" to="123v"/>: three ruled lines above,
+                                        three lines in the middle, and three lines below. The same
+                                        pattern, barely visible, is probably ruled on <locus
+                                            from="1r" to="2v"/>.</ab>
                                 </layout>
                             </layoutDesc>
                         </objectDesc>
-                        
+
                         <handDesc>
-                            <handNote xml:id="h1" script="Ethiopic">
-                           
+                            <handNote xml:id="h1" script="Ethiopic"
+                                corresp="#ms_i1 #ms_i2 #ms_i3 #ms_i4 #ms_i5 #ms_i6">
+                                <locus from="4ra" to="118v"/>
+                                <seg type="ink">Black, red</seg>
+                                <date notBefore="1800" notAfter="1868">first half of the nineteenth
+                                    century</date>
+                                <desc>Trained hand. The handwriting appears regular with broadly
+                                    spaced characters. The name of the scribe was erased in all
+                                    supplication formulars he had didicated to himself ( <locus
+                                        target="#15r '#66r #80r"/>), only the first part of his name
+                                        <foreign xml:lang="gez">ገብረ</foreign> is preserved. He is
+                                    also the main scribe of <ref type="mss" corresp="ESqma005"
+                                    />.</desc>
+                                <seg type="rubrication">Rubricated are several groups of lines on
+                                    the incipit page of <ref target="#ms_i1"/> and of all its
+                                    sections, as well as of <ref target="#ms_i2 #ms_i3 #ms_i4"/>,
+                                    regularly two lines of subsections of <ref
+                                        target="#ms_i1.1 #ms_i1.2"/> and on the incipit page of <ref
+                                        target="#ms_i5 #ms_i6"/>. Holy names (names of the members
+                                    of the Trinity are written alternating with red and black on
+                                        <locus target="#21v"/>), names of saints and holy creatures,
```

Comment:
In my understanding, names of saints and holy creatures would also fall under the category of holy names.

Reply:
I was wondering about that, I though holy names we were using "holy names" to refer to the Trinity and "sacred names" to refer to all _nomina sacra_

Reply:
Aha! I did not understand it like this at all. Good that is has come up!

Reply:
Added to agenda